.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,14 +15,14 @@ Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red
 ### How to Review
 
 1. Put
-1. the steps
-1. to verify
-1. the PR's
-1. new functionality
-1. here
+2. the steps
```

Comment:
You said "over-ruled". Are you overruling me (it should be numbered 1,2,3,4,...) or @thofmann (it should be un-changed)?